src/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To do list</title>
+    <script src="https://kit.fontawesome.com/1097d30f85.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div id="list">
+      <h2>To Do List APP</h2>
+    </div>
+</body>
```

Comment:
**Optional
Kindly make the indentation when necessary to maintain readability for any and every developer that will work on the code.